src/main/flight/pid_luxfloat.c

```diff
@@ -74,6 +75,42 @@ static const float luxITermScale = 1000000.0f / 0x1000000;
 static const float luxDTermScale = (0.000001f * (float)0xFFFF) / 512;
 static const float luxGyroScale = 16.4f / 4; // the 16.4 is needed because mwrewrite does not scale according to the gyro model gyro.scale
 
+
+// Filter coefficients, see http://www.holoborodko.com/pavel/numerical-methods/numerical-derivative/smooth-low-noise-differentiators/
+// N=2: h[0] = 1, h[-1] = -1
+// N=3: h[0] = 1/2, h[-1] = 0, h[-2] = -1/2
+// N=4: h[0] = 1/4, h[-1] = 1/4, h[-2] = -1/4, h[-3] = -1/4
+// N=5: h[0] = 5/8, h[-1] = 1/4, h[-2] = -1, h[-3] = -1/4, h[-4] = 3/8
+// N=6: h[0] = 3/8, h[-1] = 1/2, h[-2] = -1/2, h[-3] = -3/4, h[-4] = 1/8, h[-5] = 1/4
+// N=7: h[0] = 7/32, h[-1] = 1/2, h[-2] = -1/32, h[-3] = -3/4, h[-4] = -11/32, h[-5] = 1/4, h[-6] = 5/32
+// first coefficient is the divisor
+static const int nrdCoefficents2[] = { 1, 1, -1};
```

Comment:
Int has to be converted to float anyway, so store coefs as floats now
Also scaling got lost (divide coefs here)